salesagility/SuiteCRM
# Closed #1764. This fixes the localisation of suite spots.

This is a fix for the localisation of the suite spots module.  This closed issue #1764.  This is a fix, further to the points raised in pull request #1794.

## Description
This uses the key value for all enum, multi-enum and dynamic-enum fields to allow for the translation of the values.  I have tested this in US English, French and Russian.

## Motivation and Context
Required to get all of the labels / options translatable for the suite spots module / pivot grid.

## How To Test This
Pull in changes and add another language pack.  Add the appropriate labels for the suite spots module to the language file for spots.  Save / publish to dashlet a spot and switch between languages.  All enum types should be translated appropriately.

## Types of changes


### Final checklist


- [X ] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [X ] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.
